packages/cli/src/Server.ts

```diff
@@ -25,14 +25,13 @@ import axios from 'axios';
 import type { RequestOptions } from 'oauth-1.0a';
 import clientOAuth1 from 'oauth-1.0a';
 
+import type { BinaryDataService } from 'n8n-core';
```

Comment:
I think `binaryDataService` on `Server` isn't used anymore. so that line, and this import can be deleted now.